src/Umbraco.Web/Security/Providers/UmbracoMembershipProvider.cs

```diff
@@ -194,10 +194,12 @@ protected override bool PerformChangePasswordQuestionAndAnswer(string username,
         /// </returns>
         public override bool DeleteUser(string username, bool deleteAllRelatedData)
         {
-            var member = MemberService.GetByUsername(username);
+            var memberService = UmbracoContext.Current.Application.Services.MemberService;
+
+            var member = memberService.GetByUsername(username);
             if (member == null) return false;
 
-            MemberService.Delete(member);
+            memberService.WithResult().DeleteAttempt(member, UmbracoContext.Current.Security.GetUserId());
```

Comment:
This doesn't really make a difference using this here since there's now way to use the return value and since the Delete method wraps the new one anyways.